api/authorization-api-1_0.md

```diff
@@ -328,77 +329,256 @@ The following is a non-normative example of a simple Decision:
+
+`reason-description`:
+: REQUIRED. A String holding the human-readable natural language description of the reason the decision was made.
+
+The following are non-normative examples of Reason Field objects:
 
 ~~~ json
 {
-  "en": "location restriction violation"
+  "error-123": "Location restriction violation."
 }
 ~~~
-{: #reason-example title="Example Reason"}
+{: #reason-example1 title="Example Reason Field: access denial."}
 
-##### Reason Object {#reason-object}
-A Reason Object specifies a particular reason. It is a JSON object that has the following fields:
+~~~ json
+{
+  "MGR-OK": "Manager access granted."
+}
+~~~
+{: #reason-example2 title="Example Reason Field: access granted."}
 
-`id`:
-: REQUIRED. A string value that specifies the reason within the scope of a particular response.
+Putting it all together, the following is a non-normative example of a Reason Object comprising Reason Fields:
 
-`reason_admin`:
-: OPTIONAL. The reason, which MUST NOT be shared with the user, but useful for administrative purposes that indicates why the access was denied. The value of this field is a Reason Field object ({{reason-field}}).
+~~~ json
+[
+  {
+    "id": "0",
+    "reason_admin": {
+      "403": "Request failed policy C076E82F"
+    },
+    "reason_user": {
+      "403": "Insufficient privileges. Contact your administrator"
+    }
+  }
+]
+~~~
+{: #reason-example3 title="Example reason object with Reason Fields: access denied."}
 
-`reason_user`:
-: OPTIONAL. The reason, which MAY be shared with the user that indicates why the access was denied. The value of this field is a Reason Field object ({{reason-field}}).
+#### Properties context field
+OPTIONAL - The properties element is a JSON object made of custom, implementation-specific `key = value` property pairs. When provided as part of the decision response, the custom properties can provide justification, expected claims or additional information pertaining to the decision made by the PDP. The PEP can then either use these properties for further processing, or pass them along to the client for end-user interaction.
+
+A PEP can ignore this field if it doesn't know how to process it.
 
-The following is a non-normative example of a Reason Object:
+The following are a non-normative examples of custom properties response elements:
+
+~~~ json
+{
+  "group": "Employee",
+  "department": "Sales"
+}
+~~~
+{: #properties-example1 title="Example custom justification properties."}
 
 ~~~ json
 {
-  "id": "0",
-  "reason_admin": {
-    "en": "Request failed policy C076E82F"
+  "is_role_manager" : false,
+  "is_same_group": true,
+  "is_active":	true,
+  "is_within_business_hrs": true
+}
+~~~
+{: #properties-example2 title="Another example of custom justification properties."}
+
+#### s context field
```

Comment:
Nit: the section name is tructated to `s` - I don't think this is what you were trying to do